tests/test_flask_tracing.py

```diff
@@ -13,7 +13,9 @@
 test_app = app.test_client()
 
 
-tracing_all = FlaskTracing(MockTracer(FlaskScopeManager()), True, app, ["url"])
+tracing_all = FlaskTracing(
+    MockTracer(FlaskScopeManager()), True, app, ["url"], None, True
```

Comment:
this constructor gets ugly.
any pattern in Python to make it more maintainable/readable?
some config object?

Reply:
Yea, it'd be better to use named params to make it more readable and obvious.